linetools/guis/spec_widgets.py

```diff
@@ -1294,3 +1315,66 @@ def reload(self):
             #self.add_item(abssys_fil)
         self.on_list_change()
 
+
+
+# #####
+class MultiSpecWidget(QWidget):
+    """ Widget to organize a set of spectra (in one Object, e.g. XSpectrum1D)
+    """
+    def __init__(self, parent, status=None, spec_labels=None, screen_scale=1.):
+        """
+        Parameters
+        ----------
+        parent : Widget parent
+        status : Point to status bar
+        screen_scale : float, optional
+          Scale GUI dimensions
+
+        Returns
+        -------
+
+        """
+        super(MultiSpecWidget, self).__init__(parent)
+
+        # Initialize
+        self.scale = screen_scale
+        self.parent = parent  # Required
+        spec = self.parent.orig_spec
+
+        # Create the list
+        if spec_labels is None:
+            if hasattr(spec, 'labels'):
+                spec_labels = spec.labels
+            else:
+                spec_labels = ['{:d}'.format(ii) for ii in range(spec.nspec)]
```

Comment:
I think I prefer Python indexing.  That is the way
the spectra are truly indexed anyhow.